socialhome/templates/account/login.html

```diff
@@ -13,9 +13,12 @@
       <h2>{% trans "Sign In" %}</h2>
       {% get_providers as socialaccount_providers %}
       {% if socialaccount_providers %}
-      <p>{% blocktrans with site.name as site_name %}Please sign in with one
-      of your existing third party accounts. Or, <a href="{{ signup_url }}">sign up</a>
-      for a {{ site_name }} account and sign in below:{% endblocktrans %}</p>
+      <p>{% blocktrans %}Please sign in with one
+      of your existing third party accounts.{% endblocktrans %}
+        {% if is_open_for_signup %}
```

Comment:
Same here, no need to touch this file. Just need to make sure the nav bar is not rendered with the link.